runtime/crab/src/lib.rs

```diff
@@ -276,6 +276,7 @@ frame_support::construct_runtime! {
 		Assets: pallet_assets = 34,
 		Balances: pallet_balances = 5,
 		TransactionPayment: pallet_transaction_payment = 6,
+		Migrate: darwinia_migrate = 40,
```

Comment:
Index conflict with `darwinia_deposit`, xavier/staking branch

Reply:
I'm not sure which PR will be merged first. So please pay attention to the other parts. We can fix conflicts later.